myuw/templates/handlebars/card/schedule/course_cards.html

```diff
@@ -42,4 +42,8 @@ <h4>
     </div>
     {{/each}}
 </div>
+{{else}}
+<p class="text-danger"><i class="fa fa-exclamation-triangle" aria-hidden="true"></i> Some course links are not available right now. In the meantime, try  
+ <a href="https://canvas.uw.edu/" data-linklabel="Canvas LMS" style="text-decoration:underline;" target="_blank">Canvas LMS </a> or <a href="https://catalyst.uw.edu/" data-linklabel="Catalyst" style="text-decoration:underline;" target="_blank">Catalyst Web Tools</a>.</p>
```

Comment:
Should we still link to Catalyst here? Isn't it mostly disabled for participants?